src/test/admin-server/queries/CollectionAuthor.server.integration.ts

```diff
@@ -0,0 +1,211 @@
+import { gql } from 'apollo-server-express';
+import slugify from 'slugify';
+import * as faker from 'faker';
+import { CollectionAuthor } from '@prisma/client';
+import config from '../../../config';
+import { db, server } from '../';
+import { clear as clearDb, createAuthorHelper } from '../../helpers';
+import { CreateCollectionAuthorInput } from '../../../database/types';
+
+describe('queries: CollectionAuthor', () => {
+  beforeAll(async () => {
+    await clearDb(db);
+    await server.start();
+  });
+
+  afterAll(async () => {
+    await db.$disconnect();
+    await server.stop();
+  });
+
+  describe('getCollectionAuthors query', () => {
+    const GET_COLLECTION_AUTHORS = gql`
+      query getAuthors($page: Int, $perPage: Int) {
+        getCollectionAuthors(page: $page, perPage: $perPage) {
+          authors {
+            externalId
+            name
+            slug
+            bio
+            imageUrl
+            active
+          }
+          pagination {
+            currentPage
+            totalPages
+            totalResults
+            perPage
+          }
+        }
+      }
+    `;
```

Comment:
random thought maybe for a fun friday - what if we had these sample queries in their own file for easy reference for use both here and manually in the playground?

Reply:
Good idea! Moved them out to their own files (one for queries, one for mutations). This made the test suites considerably shorter and easier to navigate :).